client/landing/stepper/declarative-flow/internals/steps-repository/components/setup-form/style.scss

```diff
@@ -22,7 +22,8 @@
 	}
 
 	label,
-	textarea {
+	textarea,
+	.expanding-area textarea {
```

Comment:
/setup/link-in-bio/linkInBioSetup has a specific `.expanding-area` textarea class that gets overwritten by another font family. I don't see it testing locally, but I see it on calypso.live.